tests/contrib/hooks/test_gcp_vision_hook.py

```diff
@@ -43,8 +44,29 @@
 LOC_ID_TEST_2 = 'loc-id-2'
 PRODUCTSET_ID_TEST = 'ps-id'
 PRODUCTSET_ID_TEST_2 = 'ps-id-2'
+PRODUCTSET_NAME_TEST = 'projects/%s/locations/%s/productSets/%s' % (
```

Comment:
Nit: what do you think about changing `%` to `.format()` for consistency across GCP operators, as we discussed some time ago?